acceptance/buildpacks/manager.go

```diff
@@ -0,0 +1,53 @@
+// +build acceptance
+
+package buildpacks
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/buildpacks/pack/internal/builder"
+
+	"github.com/buildpacks/pack/testhelpers"
+)
+
+type BuildpackManager struct {
+	testObject *testing.T
+	assert     testhelpers.AssertionManager
+	sourceDir  string
+}
+
+type BuildpackManagerModifier func(b *BuildpackManager)
+
+func WithBuildpackAPIVersion(apiVersion string) func(b *BuildpackManager) {
+	return func(b *BuildpackManager) {
+		b.sourceDir = filepath.Join("testdata", "mock_buildpacks", apiVersion)
+	}
+}
+
+func NewBuildpackManager(t *testing.T, assert testhelpers.AssertionManager, modifiers ...BuildpackManagerModifier) BuildpackManager {
+	m := BuildpackManager{
+		testObject: t,
+		assert:     assert,
+		sourceDir:  filepath.Join("testdata", "mock_buildpacks", builder.DefaultBuildpackAPIVersion),
+	}
+
+	for _, mod := range modifiers {
+		mod(&m)
+	}
+
+	return m
+}
+
+type TestBuildpack interface {
+	Prepare(source, destination string) error
+}
+
+func (b BuildpackManager) PrepareBuildpacks(destination string, buildpacks ...TestBuildpack) {
```

Comment:
I'm trying to understand what "prepare" means in this context. Is the term we should be looking to use here "Install" as in it installs the provided buildpacks at the provided destination?

Reply:
Prepare is the best term I could think of for whatever an actual end user might do to get buildpacks ready for running their pack command. This could be creating a new buildpack from scratch, tarring a folder into an archive, or using pack to create a package. If I'm mapping the user action here, I don't think `install` is very descriptive for these scenarios, but happy to think about alternative terms.

Reply:
Maybe it's simply the fact that I don't associate prepare with a "finalizing" state. Prepare to me indicates that there would be additional steps after the fact to get it into its final desired state. That doesn't seem to be the case here. In either way, I wouldn't say it's a blocker.